src/mumble/GlobalShortcut_win.cpp

```diff
@@ -202,6 +202,20 @@ void GlobalShortcutWin::run() {
 	pDI->Release();
 }
 
+bool GlobalShortcutWin::event(QEvent *event) {
+	QEvent::Type type = event->type();
+	if (type == INJECTKEYBOARDMESSAGE_QEVENT) {
```

Comment:
We should check if winhooks are enabled here, and avoid handling the events if not.